catalog/solr/catalog-solr-core/src/test/java/ddf/catalog/source/solr/SolrFilterDelegateTest.java

```diff
@@ -141,17 +145,19 @@ public void polygonWithHoleAndSelfIntersecting() {
   }
 
   @Test
-  public void multiPolygon() {
+  public void multiPolygon() throws ParseException {
     String wkt =
         "MULTIPOLYGON (((30 20, 45 40, 10 40, 30 20)), ((15 5, 40 10, 10 20, 5 10, 15 5)))";
     when(mockResolver.getField(
             "testProperty", AttributeFormat.GEOMETRY, false, Collections.emptyMap()))
         .thenReturn("testProperty_geohash_index");
     SolrQuery query = toTest.contains("testProperty", wkt);
+    MultiPolygon multiPolygon = (MultiPolygon) new WKTReader().read(wkt);
+    // Need to use union since allowMultiOverlap is enabled
+    MultiPolygon unioned = (MultiPolygon) new UnaryUnionOp(multiPolygon).union();
```

Comment:
❓ Is this new union requirement going to impact geos read in from other bundles in the system?

Reply:
Maybe I need to reword this comment. Nothing new is required of WKTs being read by this bundle. When `allowMultiOverlap` is enabled, spatial4j will automatically compute the union of any `GeometryCollection` instance. This comment is just saying that we need to compute the union of the expected WKT for the assertion.